### PR TITLE
Clearer instructions on using NGINX OSS vs NGINX Plus

### PR DESCRIPTION
**Pull request by wallrj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,11 +223,22 @@ spec:
...
 ```
 
+Check that the Issuer has connected to TPP using:
+
+```console
+kubectl describe issuer venafi-tpp-issuer
+```
+
+You should see a message `Verified issuer with Venafi server`.
+If not, check for error messages and check that you have used the correct TPP credentials and URL.
```
Comment (meyskens) [inline README.md:240]: The user also might have to wait a few seconds, we should note that

**Comment by meyskens** [comment]
Looks good! Just one tiny comment

File: README.md (hunk 1 above)
Comment (wallrj) [reply README.md:242]: Fixed.

**Pull request merged**